.github/workflows/tag.yaml

```diff
@@ -0,0 +1,52 @@
+name: Bump Git/Docker Version
+on:
+  push:
+    branches:
+      - master
+env:
+  PLATFORMS: linux/arm/v7,linux/amd64,linux/arm64, windows/amd64, darwin/amd64
+
+jobs:
+  semver:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.tagging.outputs.new_tag }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.39.0
+      id: tagging
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        WITH_V: true
+        RELEASE_BRANCHES: master
+        DEFAULT_BUMP: minor
+  docker:
+    runs-on: ubuntu-latest
+    needs: semver
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Do we really need this here? We should be able to cross-compile and copy binaries into containers for different architectures without QEMU?

Reply:
Yeah that's fair. Multiple ways to do it. I've removed it for now.